### PR TITLE
fix holding ratio related error

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: Trading/Mode/daily_trading_mode/daily_trading_mode.py [hunk 1]
```diff
@@ -213,9 +213,17 @@ def _get_market_quantity_from_risk(self, eval_note, quantity, quote, selling=Fal
         checked_factor = check_factor(self.QUANTITY_MARKET_MIN_PERCENT, self.QUANTITY_MARKET_MAX_PERCENT, factor)
         return checked_factor * quantity
 
+    async def _get_ratio(self, currency):
+        try:
+            return await self.get_holdings_ratio(currency)
+        except KeyError:
+            # Can happen when ref market is not in the pair, data will be available later (ticker is now registered)
+            # consider a high ratio not to take too much risk and do not prevent order creation
+            return 0.35
```
Comment (Herklos) [inline Trading/Mode/daily_trading_mode/daily_trading_mode.py:222]: How did you choose this value ?

Comment (Herklos) [reply Trading/Mode/daily_trading_mode/daily_trading_mode.py:222]: Can we use a constant for this value please

Comment (GuillaumeDSM) [reply Trading/Mode/daily_trading_mode/daily_trading_mode.py:222]: > How did you choose this value ?

This allows to keep going with the trading without blocking it, 0.35 is a middleground bewteen taking unecessary risks and cancelling the signal by using a too big ratio

Comment (Herklos) [reply Trading/Mode/daily_trading_mode/daily_trading_mode.py]: Ok :+1:

**Pull request merged**